src/fullnode.rs

```diff
@@ -57,7 +57,7 @@ impl Fullnode {
         sigverify_disabled: bool,
     ) -> Self {
         info!("creating bank...");
-        let bank = Bank::new_default(leader_addr.is_some());
+        let bank = Bank::new_default(leader_addr.is_none());
```

Comment:
can we remove this code?  does it serve any purpose except for metrics?

Reply:
How about under a separate ticket? This bugfix is correct, so we should at least merge this one asap.